README.md

```diff
@@ -1 +1,162 @@
-# federation-support
\ No newline at end of file
+# federation-support
+
+A module to use the federation feature in `mercurius`.
+
+This module extract the federation parts from `mercurius` to be used as external plugin.
+
+## Quick start
+
+```js
+'use strict'
+
+const Fastify = require('fastify')
+const mercurius = require('mercurius')
+const { createGateway, buildFederationSchema } = require('/index')
```

Comment:
What name are we going to use for this modulev

Reply:
Some proposal

gateway
* `@mercurius/gateway`
* `@mercurius/apollo-gateway`

schemabuilder
* `@mercurius/federation-schema`
* `@mercurius/apollo-federation-schema`